resources/views/software_engineer.blade.php

```diff
@@ -0,0 +1,136 @@
+                        <x-icons.arrow></x-icons.arrow>
+                        {{__("content.software_engineer.section_1.content_1")}}
+                    </li>
+                    <li class="flex items-start sm:items-center text-left leading-relaxed gap-x-4">
+                        <x-icons.arrow></x-icons.arrow>
+                        {{__("content.software_engineer.section_1.content_2")}}
+                    </li>
+                    <li class="flex items-start sm:items-center text-left leading-relaxed gap-x-4">
+                        <x-icons.arrow></x-icons.arrow>
+                        {{__("content.software_engineer.section_1.content_3")}}
+                    </li>
+                    <li class="flex items-start sm:items-center text-left leading-relaxed gap-x-4">
+                        <x-icons.arrow></x-icons.arrow>
+                        {{__("content.software_engineer.section_1.content_4")}}
+                    </li>
+                    <li class="flex items-start sm:items-center text-left leading-relaxed gap-x-4">
+                        <x-icons.arrow></x-icons.arrow>
+                        {{__("content.software_engineer.section_1.content_5")}}
+                    </li>
+                    <li class="flex items-start sm:items-center text-left leading-relaxed gap-x-4">
+                        <x-icons.arrow></x-icons.arrow>
+                        {{__("content.software_engineer.section_1.content_6")}}
+                    </li>
+                    <li class="flex items-start sm:items-center text-left leading-relaxed gap-x-4">
+                        <x-icons.arrow></x-icons.arrow>
+                        {{__("content.software_engineer.section_1.content_7")}}
+                    </li>
+                    <li class="flex items-start sm:items-center text-left leading-relaxed gap-x-4">
+                        <x-icons.arrow></x-icons.arrow>
+                        {{__("content.software_engineer.section_1.content_8")}}
+                    </li>
+                    <li class="flex items-start sm:items-center text-left leading-relaxed gap-x-4">
+                        <x-icons.arrow></x-icons.arrow>
+                        {{__("content.software_engineer.section_1.content_9")}}
+                    </li>
+                </ul>
+            </div>
+        </div>
+        <div class="basis-2/5 3xl:basis-1/2 self-center relative z-10">
+            <img src="{{ asset('graphics/web-1.png') }}" class="h-auto w-full aspect-auto object-center shrink-0"
+                 alt="placeholder"/>
+        </div>
+    </section>
+    <section class="bg-gradient-to-br from-gray-light to-white">
+        <x-waves.wave-1.top></x-waves.wave-1.top>
+        <div class="flex flex-col 2xl:flex-row px-[10%] md:px-[20%] pt-32 gap-16 md:gap-24">
+            <div class="basis-2/5 self-center">
+                <img src="{{ asset('graphics/web-1.png') }}" class="h-auto w-full aspect-auto object-center shrink-0"
```

Comment:
Description of `alt` attribute should be more verbose. But we can do that in separated task - `SEO` stuff.

Reply:
It'll be done in #235